lib/gui/os/window-progress.js

```diff
@@ -46,7 +47,8 @@ exports.currentWindow = electron.remote.getCurrentWindow();
 exports.set = (percentage, status) => {
   exports.currentWindow.setProgressBar(utils.percentageToFloat(percentage));
   const statusString = status === 'check' ? 'Validating' : 'Flashing';
-  exports.currentWindow.setTitle(`Etcher – ${statusString} ${percentage}%`);
+  const dash = '\u2013';
```

Comment:
It's more aligned with typography, where hyphens are used to link words together, and dashes used to separate words alike commas or parentheses.